draft/spec/index.html

```diff
@@ -876,21 +885,26 @@ <h2>Versioned OCFL Object</h2>
     ├── 0=ocfl_object_1.0
     ├── inventory.json
     ├── inventory.json.sha512
-    ├── v1
-    │   ├── empty.txt
-    │   ├── foo
-    │   │   └── bar.xml
-    │   ├── image.tiff
+    ├── v1        
     │   ├── inventory.json
-    │   └── inventory.json.sha512
-    ├── v2
-    │   ├── foo
-    │   │   └── bar.xml
+    │   ├── inventory.json.sha512
+    │   └── content
+    │       ├── empty.txt
+    │       ├── foo
+    │       │   └── bar.xml
+    │       └── image.tiff
+    ├── v2    
     │   ├── inventory.json
-    │   └── inventory.json.sha512
+    │   ├── inventory.json.sha512
+    │   └── content
+    │       ├── foo
+    │       └── bar.xml
     └── v3
         ├── inventory.json
-        └── inventory.json.sha512
+        ├── inventory.json.sha512
+        └── content
```

Comment:
We are suggesting not to include empty `content` directories.

Reply:
fixed